polars/polars-io/src/csv/buffer.rs

```diff
@@ -29,27 +29,27 @@ impl PrimitiveParser for Float64Type {
     }
 }
 
```

Comment:
Can you remove these changes from git? I would not expect any changes in `buffer.rs` for this fix.

Reply:
Oops, my bad!